bedrock/firefox/templates/firefox/new/trailhead/thanks-b.html

```diff
@@ -31,68 +29,58 @@
   <div class="mzp-l-content">
     <div class="mzp-c-notification-bar mzp-t-warning unsupported">
       <p>
-        {% trans url='https://support.mozilla.org/kb/end-support-windows-xp-and-vista' %}
-        You’re using an insecure, outdated operating system <a href="{{ url }}">no longer supported by Firefox</a>.
-        {% endtrans %}
+        You’re using an insecure, outdated operating system <a href="https://support.mozilla.org/kb/end-support-windows-xp-and-vista">no longer supported by Firefox</a>.
```

Comment:
Note: this is an experimental page and has not been exposed for L10n. As such, I'm removing the `_('')` and `trans` calls from the template.